deploy/README.md

```diff
@@ -29,3 +29,4 @@ Most environment variables can be found in [common/env.sh](common/env.sh).
 | `REGISTRY_USERNAME`     | `string`                   | Set docker registry username to pull the docker.io/stackrox/main image. |
 | `REGISTRY_PASSWORD`     | `string`                   | Set docker registry password to pull the docker.io/stackrox/main image.  |
 | `STORAGE`               | `none`  \| `pvc`           | Defines which storage to use for the Central database, to preserve data between Central restarts it is recommended to use `pvc`.                                                |
+| `SENSOR_DEV_RESOURCES`  | `true`  \| `false`                                                                 | (defaults to `true`) When set to true, Sensor will be deployed with reduced memory/cpu requests. This should be used exclusively for testing and development environments.
```

Comment:
Spacing looks odd and it is missing the trailing `|` for the table.

Reply:
Pushing a fix for this, looks like other lines above were also not rendering the table as it should